src/components/element-editor/basic-details/basic-details.tsx

```diff
@@ -69,6 +70,7 @@ export const BasicDetails = () => {
                                     shouldDirty: true,
                                 });
                                 setValue("validation", []);
+                                clearErrors("validation");
```

Comment:
replace with `resetField`

side note: remember to move this logic to a useEffect as proposed in https://github.com/LifeSG/web-form-builder/pull/106#discussion_r1708421548